src/Middleware/Session/src/NoOpSessionStore.cs

```diff
@@ -13,7 +14,7 @@ internal class NoOpSessionStore : IDictionary<EncodedKey, byte[]>
         {
             get
             {
-                return null;
+                return null!;
```

Comment:
💡 It seems better to change the interface type to `IDictionary<EncodedKey, byte[]?>`

Reply:
Perhaps. My guess is that this does method does not get called if the session is unavailable but I can't be certain.

Reply:
This could be called if someone didn't check IsAvailable.

Reply:
I was able to abstract this contract a little better and get rid of some these oddities here.